frontend/src/Components/Page/Toolbar/PageToolbarButton.css

```diff
@@ -2,7 +2,8 @@
   composes: link from '~Components/Link/Link.css';
 
   padding-top: 4px;
-  width: $toolbarButtonWidth;
+  width: min-content;
+  min-width: $toolbarButtonWidth;
```

Comment:
Failing lint with the following:
`Expected "min-width" to come before "width"  order/properties-order`

Reply:
Sorry. I forgot to run stylelint